nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common-base/nuxeo.defaults

```diff
@@ -273,3 +273,39 @@ nuxeo.cors.allowSubdomains=true
 nuxeo.cors.supportedMethods=GET,POST,HEAD,OPTIONS,DELETE,PUT
 nuxeo.cors.urls=
 nuxeo.cors.pattern=/.*
+
+# Bulk Service Action concurrency and partitions
+nuxeo.bulk.action.attachRetentionRule.defaultConcurrency=2
+nuxeo.bulk.action.attachRetentionRule.defaultPartitions=4
+nuxeo.bulk.action.automation.defaultConcurrency=2
+nuxeo.bulk.action.automation.defaultPartitions=2
+nuxeo.bulk.action.csvExport.defaultConcurrency=2
+nuxeo.bulk.action.csvExport.defaultPartitions=2
+nuxeo.bulk.action.driveFireGroupUpdatedEvent.defaultConcurrency=2
+nuxeo.bulk.action.driveFireGroupUpdatedEvent.defaultPartitions=4
```

Comment:
4?

Reply:
this is the same default as before, just set explicitly